firmware/controllers/idle_thread.cpp

```diff
@@ -212,19 +232,28 @@ static msg_t ivThread(int param) {
 		undoIdleBlipIfNeeded();
 
 		float clt = engine->sensors.clt;
-		float cltCorrection = cisnan(clt) ? 1 : interpolate2d("cltT", clt, config->cltIdleCorrBins, config->cltIdleCorr,
-		CLT_CURVE_SIZE) / PERCENT_MULT;
+		bool isRunning = engine->rpmCalculator.isRunning(PASS_ENGINE_PARAMETER_SIGNATURE);
+		float cltCorrection;
+		if (cisnan(clt))
+			cltCorrection = 1.0f;
+		// Use separate CLT correction table for cranking
+		else if (engineConfiguration->overrideCrankingIacSetting && !isRunning)
```

Comment:
please use curly brackets for complex if statements

to be honest I am starting to think that any if statement should have curly brackets